### PR TITLE
Update collections.yml of cutting edge technologies

### PR DESCRIPTION
**Pull request by hncslwx**
Remove “merging technologie 新兴技术”, 
ChatGPT will be moved to AI class.

Adding “cutting edge technologies 前沿技术” 
Tier 2 yml files areready.

### CODE REVIEW

File: collections.yml [hunk 1]
```diff
@@ -43,11 +43,17 @@
...
     - continuous-integration-tools # 持续集成工具
-- ident: emerging-technologie
-  name: Emerging Technologie
-  name_cn: 新兴技术
+
+- ident: cutting-edge-technologies
+  name: Cutting Edge Technologies
+  name_cn: 前沿技术
   items:
-    - chatgpt # ChatGPT
```
Comment (eyehwan) [inline collections.yml:50]: Why do you remove this item?

Comment (hncslwx) [reply collections.yml:50]: Actually it not a remove, it is changing name from "merging technologie 新兴技术" to "cutting edge technologies 前沿技术", which we put in more classes into it.

Comment (eyehwan) [reply collections.yml:50]: I mean this one:
- chatgpt # ChatGPT

Comment (hncslwx) [reply collections.yml:50]: chatgpt will move to AI

**Review by eyehwan — APPROVED** [review]

**Pull request merged**